pkg/option/config.go

```diff
@@ -2649,23 +2649,23 @@ func (c *DaemonConfig) calculateDynamicBPFMapSizes(totalMemory uint64, dynamicSi
 	if !viper.IsSet(CTMapEntriesGlobalTCPName) {
 		c.CTMapEntriesGlobalTCP =
 			getEntries(CTMapEntriesGlobalTCPDefault, LimitTableMin, LimitTableMax)
-		log.Debugf("option %s set by dynamic sizing to %v (default %v)",
+		log.Infof("option %s set by dynamic sizing to %v (default %v)",
```

Comment:
What does the `default (...)` section of these log messages mean if the default is to dynamically size the maps....?

Reply:
Maybe something like `static default size` would be more appropriate now?

Reply:
I would find it confusing if I deploy Cilium without thinking about this option, then read through the logs and see that it configured the setting to some value then told me the default is a different value.

What are we trying to communicate with that value? Is it a suggested value or something else? Maybe this would be more appropriate on the documentation page for maps & sizes?

Reply:
Good point. These "default" values were mainly useful for debugging during development, so I think it would be fine to drop them from the log message now. Also, these values are already documented in the docs, it seems: https://docs.cilium.io/en/latest/architecture/?highlight=scale#bpf-map-limitations